### PR TITLE
Not assosciated bug

### PR DESCRIPTION
**Pull request by JudahSchwartz**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://github.com/demisto/etc/issues/47863)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/Scripts/IncidentFields/TestData/integration_incidentfields.json [hunk 1]
```diff
@@ -1 +1 @@
...ive": true, "name": "Groups", "ownerOnly": false, "required": false, "modified": "2019-12-30T16:43:18.35567477-05:00", "columns": null, "selectValues": null, "validationRegex": "", "sla": 0}, {"sortValues": null, "closeForm": false, "cliName": "hostname", "neverSetAsRequired": false, "threshold": 72, "defaultRows": null, "id": "indicator_hostname", "validatedError": "", "group": 2, "script": "", "isReadOnly": false, "commitMessage": "", "system": false, "content": true, "unsearchable": false, "version": 4, "unmapped": false, "hidden": false, "type": "shortText", "shouldCommit": false, "editForm": true, "fieldCalcScript": "", "description": "", "systemAssociatedTypes": null, "breachScript": "", "vcShouldIgnore": false, "associatedTypes": ["IP", "hostname"], "locked": false, "placeholder": "", "useAsKpi": false, "associatedToAll": false, "prevName": "Hostname", "caseInsensitive": true, "name": "Hostname", "ownerOnly": false, "required": false, "modified": "2019-12-30T16:43:18.15837464-05:00", "columns": null, "selectValues": null, "validationRegex": "", "sla": 0}, {"sortValues": null, "closeForm": false, "cliName": "id", "neverSetAsRequired": false, "threshold": 72, "defaultRows": nu
```
Comment (dantavori) [inline Packs/CommonScripts/Scripts/IncidentFields/TestData/integration_incidentfields.json:1]: what is the change in this file?

File: Packs/CommonScripts/ReleaseNotes/1_6_52.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Scripts
+##### IncidentFields
+- Fixed an error where this script would fail if incident fields were not properly configured.
+- Added README.md to IncidentFields
```
Comment (dantavori) [inline Packs/CommonScripts/ReleaseNotes/1_6_52.md:5]:
```suggestion
```
no need, in RNs we only put changes that are visible to the customers through the marketplace UI.

**Review by dantavori — APPROVED** [review]
niiice 👍🏽

File: Packs/CommonScripts/Scripts/IncidentFields/TestData/integration_incidentfields.json (hunk 1 above)
Comment (JudahSchwartz) [reply Packs/CommonScripts/Scripts/IncidentFields/TestData/integration_incidentfields.json:1]: for all of these I went and found a pack (chose detectionsla randomly) and switched it to the case that caused the script to fail

Comment (JudahSchwartz) [reply Packs/CommonScripts/Scripts/IncidentFields/TestData/integration_incidentfields.json:1]: so I switched assosciatedTypes to none and assosciateAll to false

**Pull request merged**